dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -457,8 +457,8 @@ BEGIN
     EDITTEXT 1404, 19, 120, 193, 14, ES_AUTOHSCROLL
     LTEXT "F&acilități:", 1409, 19, 138, 182, 8, NOT WS_GROUP
     EDITTEXT 1403, 19, 149, 193, 14, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", 1, 100, 189, 60, 14
-    PUSHBUTTON "A&nulează", 2, 164, 189, 60, 14
+    DEFPUSHBUTTON "&OK", 1, 100, 189, 60, 14
+    PUSHBUTTON "&Revocare", 2, 164, 189, 60, 14
```

Comment:
This change introduces an accelerator collision with the line
 LTEXT "&Rețea:", 1410, 9, 35, 213, 8, NOT WS_GROUP
Please check your romanian Windows to find out which of both lines need to be changed.